webpack.prod.js

```diff
@@ -15,6 +17,27 @@ module.exports = mergeWithCustomize({
     new MiniCssExtractPlugin({
       filename: "[name].[contenthash].css",
     }),
+    new CompressionPlugin({
```

Comment:
do we need both gzip and brotli?

Reply:
let's keep them I guess for a while and we can remove one of them in the future. Merge away!